src/ScriptCs.Contracts/IConsole.cs

```diff
@@ -10,7 +10,7 @@ public interface IConsole
 
         void WriteLine(string value);
 
-        string ReadLine();
+        string ReadLine(string prompt);
```

Comment:
Good point. The code we had before should work / we don't need prompt to be a param.